Adding note about enabling Git LFS

jchu231 opened the pull request.

mikeseese reviewed (changes requested):
Could you move this warning to the top of the readme? There needs to be a warning that people can't use GitHub's "Download ZIP" option and must clone with LFS installed. Maybe make a `#` or `##` header and a ⚠️ emoji?

jchu231 commented:
> Could you move this warning to the top of the readme? There needs to be a warning that people can't use GitHub's "Download ZIP" option and must clone with LFS installed. Maybe make a `#` or `##` header and a ⚠️ emoji?

Updated to add note at the top and also added the link to Git LFS
